src/audio/.gitignore

```diff
@@ -23,7 +23,6 @@
 /so_locations
 /spd_alsa.la
 /spd_libao.la
-/spd_nas.la
```

Comment:
This is also a spurious removal, it'd rather be adding `/spd_pipewire.la` that would make sense